internal/cutil/aliases.go

```diff
@@ -26,3 +43,13 @@ type SizeTPtr unsafe.Pointer
 
 // FreeFunc is a wrapper around calls to, or act like, C's free function.
 type FreeFunc func(unsafe.Pointer)
+
+//
+// Functions
```

Comment:
If the file is getting `// SECTION` type comments it might be better just to split this into multiple files. Go's pretty good in that it makes that kind of splitting nice and easy.

Reply:
Seriously? That's what I had first, and found it silly, so I moved everything into one file. :smirk:

Reply:
Also I would argue, that Go is goo d with that, but Cgo not so much. It will generate and compile several files for a two line constant_aliases.go file. How about splitting it up when it indeed gets big?

Reply:
OK, but could we drop the "section header comments"? I find them to be more visual noise than helpful.

Reply:
Sure